daemon/CMakeLists.txt

```diff
@@ -54,8 +54,12 @@ foreach(P daemon)
     endif()
 
     if (WITH_SYSTEMD)
+        pkg_get_variable(SYSTEMD_SYSTEM_UNIT_DIR
+            systemd systemdsystemunitdir
+            DEFINE_VARIABLES prefix=)
```

Comment:
`DEFINE_VARIABLES` was added in CMake 3.28, we're declaring support for 3.12 in top-level CMakeLists.txt.